File: 1 Semestre/Analise e Projeto de Algoritmos (Mauricio)/Aula 4 Estruturas de Controle (Selecao)/Exercicio 4.py
'''4) Idem ao 2) porém com três valores. '''

a = int(input("Digite o primeiro valor: "))
b = int(input("Digite o segundo valor: "))
c = int(input("Digite o terceiro valor: "))

if a > b and a > c: print("O primeiro valor e o maior!")
elif b > c: print("O segundo valor e o maior!")
else: print("O terceiro valor e o maior!")

File: 2 Semestre/Sistemas Embarcados (C)/Aula 4/Exercicio 4.c
#include <stdio.h>

int main(){
    int a, b;

    printf("Digite o primeiro valor: ");
    scanf("%i",&a);

    printf("Digite o segundo valor: ");
    scanf("%i",&b);

    a = a*2;
    b = b*2;

    printf("Primeiro valor - %i\nSegundo valor - %i",a,b);}